README.md

```diff
@@ -1,2 +1,83 @@
-# puppetlabs-powershell_task_helper
-A helper library for Puppet Tasks written in PowerShell
+# powershell_task_helper
+
+A helper library for Puppet Tasks written in PowerShell.
+
+## Table of Contents
+
+1. [Description](#description)
+1. [Setup - The basics of getting started with powershell_task_helper](#setup)
+    * [Beginning with powershell_task_helper](#beginning-with-powershell_task_helper)
+1. [Usage - Configuration options and additional functionality](#usage)
+1. [Limitations - OS compatibility, etc.](#limitations)
```

Comment:
```suggestion
```

This section doesn't have content under it, so the ToC link should be removed if the section is.